tests/run_all_tests.sh

```diff
@@ -1,5 +1,22 @@
 #!/bin/bash
 
+# Download TFDS metadata to flax/.tdfs/metadata directory. 
```

Comment:
Let's extract this code into a separate script called `download_dataset_metadata.sh`

Reply:
Done.